main_test.go

```diff
@@ -221,7 +221,7 @@ func Example_makeASimpleTransfer() {
 		panic(err)
 	}
 
-	key, err := types.CreateStorageKey(meta, "System", "AccountNonce", signature.TestKeyringPairAlice.PublicKey, nil)
```

Comment:
It is not clear to me what's been requested here. 
The result of the key here is used to set the nonce, a uint32.
Same below. Could you elaborate a little more?

Reply:
In the past yes, the AccountNonce returned a uint32, since now we are using Account, which will return an AccountInfo object https://github.com/centrifuge/go-substrate-rpc-client/blob/master/types/account_data.go#L20
Then you need to adapt the test to use that struct instead of a uint32. See this whole test case I posted, already does that: https://github.com/centrifuge/go-substrate-rpc-client/blob/master/teste2e/author_submit_extrinsic_test.go#L30

Reply:
Ok, now I see, I think I ve fixed this.